content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -489,6 +489,11 @@ If a file is created and deleted, but has an open file descriptor,
 it continues to consume space. Quota tracking records that space accurately
 whereas directory scans overlook the storage used by deleted files.
 
+To use quotas to track a pod's resource usage, the pod must be in 
+a user namespace. Within user namespaces, the kernel restricts changes 
+to projectIDs on the filesystem, ensuring the reliability of storage 
+metrics calculated by quotas.
+
 If you want to use project quotas, you should:
 
 * Enable the `LocalStorageCapacityIsolationFSQuotaMonitoring=true`
```

Comment:
Does this still need enabling? I understand it's now on by default.

Reply:
actually I've proposed it not be https://github.com/kubernetes/kubernetes/pull/126355

Reply:
Either way, let's make sure things are consistent within this PR. _And_ that they are accurate.